Code/Mantid/MantidQt/CustomInterfaces/src/Indirect/ApplyCorr.cpp

```diff
@@ -102,16 +102,7 @@ namespace IDA
     {
       pyInput += "useCor = True\n";
       QString corrections = m_uiForm.dsCorrections->getCurrentDataName();
-      if ( !Mantid::API::AnalysisDataService::Instance().doesExist(corrections.toStdString()) )
-      {
-        pyInput +=
-          "corrections = loadNexus(r'" + m_uiForm.dsCorrections->getFullFilePath() + "')\n";
-      }
-      else
-      {
-        pyInput +=
-          "corrections = '" + corrections + "'\n";
-      }
+      pyInput += "corrections = '" + corrections + "'\n";
```

Comment:
This is no longer needed as the corrections file is loaded in the UI (the error was the ```loadNexus``` function no longer exists).